src/Microsoft.ML.OnnxTransform/OnnxUtils.cs

```diff
@@ -4,27 +4,26 @@
 
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.Internal.Utilities;
-using Microsoft.ML.Scoring;
+using Microsoft.ML.OnnxRuntime;
+using System.Numerics.Tensors;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using Microsoft.ML.StaticPipe;
 
-using OnnxShape = System.Collections.Generic.List<long>;
-using Microsoft.ML.Data;
+using OnnxShape = System.Collections.Generic.List<int>;
 
 namespace Microsoft.ML.Transforms
 {
     /// <summary>
-    /// OnnxModel is a facad for ModelManager. ModelManager is provided by Sonoma API,
-    /// and it has a lot of functionality (multiple models, multiple versions) that are not
-    /// needed by Onnx transform, which only needs a single model. This facad simplifies the
-    /// usage of onnx model.
+    /// OnnxModel is a utility class to load ONNX models and retrieve metadata
+    /// for inputs and outputs. The metadata includes the names, shapes and types
+    /// It provides API to open a session, score tensors (NamedOnnxValues) and return
+    /// the results.
     /// </summary>
-    internal sealed class OnnxModel
+    public sealed class OnnxModel
     {
+
         /// <summary>
         /// OnnxModelInfo contains the data that we should get from
         /// Sonoma API once that functionality is added.
```

Comment:
Nothing in here should reference `Sonoma` anymore, right?

Reply:
Fixed.

---
In reply to: [236336917](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236336917) [](ancestors = 236336917)